instrumentation/opentelemetry-api/opentelemetry-api-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/OpenTelemetryApiInstrumentationModule.java

```diff
@@ -24,6 +25,8 @@ public List<TypeInstrumentation> typeInstrumentations() {
         new ContextInstrumentation(),
         new ContextStorageWrappersInstrumentation(),
         new OpenTelemetryInstrumentation(),
-        new SpanInstrumentation());
+        new SpanInstrumentation(),
+        // instrumentation-api specific instrumentation
+        new HttpRouteStateInstrumentation());
```

Comment:
The solution to this issue turned out to be a bit more complex than expected -- I don't particularly like this, but I had to insert additional instrumentation-api instrumentation into the opentelemetry-api module, cause it uses the `AgentContextWrapper`. It doesn't add any muzzle dependencies though, so it should be completely neutral for applications that just use opentelemetry-api without instrumentation-api.